docusaurus/docs/users/howto/self-hosting.md

```diff
@@ -4,41 +4,55 @@ sidebar_position: 5
+Since defining all the parts of a server configuration can be time consuming, we put together a [pre-made server configuration](https://github.com/BobaBoard/selfhost-example/) you can clone for your own low-maintenance, easily-reproducible BobaBoard server. To make use of our config, you'll need to install [NixOS](https://nixos.org/) on an unconfigured server and make a few tweaks to connect your server to a domain you control instead of to one we control.
 
 To follow these instructions, you will need:
 * A GitHub account *(and the knowledge of how to clone a repository)*
 * VSCode
 * An unconfigured server
 
-So far, there are two examples of self-hosting—one hosted by the BobaBoard organization [here](https://github.com/BobaBoard/selfhost-example), and one created during a livestream [here on Ms Boba's personal account](https://github.com/essential-randomness/boba-test) to simulate an outsider hosting a BobaBoard server from scratch.
-
 In our self-hosting demo livestream, Ms Boba showed the viewer how to install NixOS on a [Hetzner](https://www.hetzner.com/cloud/) VPS (virtual private server). While the instructions provided below assume you're also using Hetzner, it's not required! Other server options include [DigitalOcean](https://www.digitalocean.com/products/droplets/), [Vultr](https://www.vultr.com/), and even your own physical computer in your home (if you don't mind leaving your computer on 100% of the time!)
 
 ## Creating a Server
 
 In your Hetzner dashboard, navigate to the "Create a server" screen. You will need to indicate a choice in each of the following categories:
 * Server name
-* Region *(Up to you!)*
-* Image/Operating system *(Our suggestion: Ubuntu)*
-* Virtual CPU type *(BobaBoard doesn't need many CPU resources to run, so you can choose to use only a small number of vCPUs with 4 GB ram.)*
-* Network address options *(Our suggestion: Both Public IPv4 & Public IPv6.)*
-* SSH key(s) *(If you don't already have an OpenSSH key you want to use, [generate one](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).)*
+* Region 
+:::tip
+This is where your new server will live, not where *you* live.
+:::
+* Image/Operating system 
+:::tip
+Our suggestion: Ubuntu.
+:::
+* Virtual CPU type 
+:::tip
+BobaBoard doesn't need much CPU usage during everyday use, but setting it up goes very slowly if you pick the cheapest option. Our suggestion: Pay for a bit of higher-tier CPU use, then drop to a lower tier once you're all set up. All you'd have to redo would be [this step](#getting-files-from-the-new-server).
+:::
+* Network address options 
+:::tip
+Our suggestion: Both Public IPv4 & Public IPv6.
+:::
+* SSH key(s) 
+:::tip
+If you don't already have an OpenSSH key you want to use, [generate one](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
```

Comment:
"If you don't already have an SSH key you want to use, you can follow GitHub's instructions to generate one."